backend/database-router.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env mocha */
+
+import assert from 'node:assert'
+import RouterBackend from './database-router.js'
+import { cloneDeep, omit } from 'turtledash'
+
+const validConfig = {
+  'abc': {
+    name: 'sqlite',
+    options: {
+      dirname: './data'
+    }
+  },
+  '*': {
+    name: 'fs',
+    options: {
+      dirname: './data'
+    }
+  }
+}
+
+describe('DatabaseRouter::validateConfig', () => {
```

Comment:
I think we could use `describe` and `it` from `node:test`, which will reduce the need to have extra dependencies.